lib/src/main/kotlin/com/swmansion/starknet/data/types/transactions/Transaction.kt

```diff
@@ -99,7 +99,7 @@ data class InvokeTransaction(
 
     override val type: TransactionType = TransactionType.INVOKE,
 ) : Transaction() {
-    internal fun toPayload(): InvokeFunctionPayload {
+    fun toPayload(): InvokeFunctionPayload {
         val invocation = Call(
             contractAddress = contractAddress,
             calldata = calldata,
```

Comment:
Perhaps we should add a comment indicating that this function is not exactly a part of the stable API? 

cc @Solpatium

Reply:
What does it change for the users? If we need to remove it we'll deprecate it first and offer an alternative anyway.

Reply:
Maybe let's not expose it at all then and consider alternative approach now - because it doesn't make sense to me to expose it just to have it removed 2 versions from now (and deprecations aren't really practical at this stage of the development).

Reply:
If I remember correctly, this is one of the functions mentioned specifically by someone from the community. I think I'd just expose it anyway now, and deprecate/remove later on. Is that ok with you guys?

Reply:
It is ok for me, the functionality here is straightforward. In my opinion the problem is when deprecating something that touches a lot other code.